app/models/federated_protocols/saml.rb

```diff
@@ -25,7 +25,7 @@ def acr_values
     end
 
     def vtr
-      nil
+      [request.requested_vtr_authn_context]
```

Comment:
Would there be concern about breaking the interface from changing to requiring an array?

Reply:
The user facing SAML interface? I believe it depends on how we plan to consume the array of VTRs. If the VTR is expressed as a series of AuthnContext nodes ([as has been suggested](https://gsa-tts.slack.com/archives/CNCGEHG1G/p1709157024425319?thread_ts=1709156469.748839&cid=CNCGEHG1G)) then this approach should be compatible. If the VTR is expressed as a space or comma separated list within an AuthnContext node then this should still be compatible. If the VTR is expressed as a JSON list this would be tricky to make compatible because it would have to recognize a bare VTR or a JSON encoded list of VTRs.

Reply:
Ah, got it, thanks!

I'm not necessarily worried about uses of the `sam_idp` gem outside of ours either.